docs/hydra/guides/deploy-hydra-example.mdx

```diff
@@ -0,0 +1,494 @@
+- **SSD**: 25Gb
+- **VPC network**: default
+- **Authentication**: SSH Keys. Don't forget to add your own SSH key
+- **Region**: Choose your region
+
+:::note
+
+This example shows a single configuration on a small droplet. The configuration of a virtual machine (VM) may vary depending on
+the scale of your application.
+
+:::
+
+Wait for the virtual machine (VM) to start up and copy the IP address.
+
+This guide uses `oauth2.example.com` as a hostname to run Ory Hydra. Replace it with `<something>.<your-domain>` for your
+purposes. You need to configure DNS, create an `A type` record, and point it to the VM's IP.
+
+## Install required dependencies
+
+Connect to your [droplet via SSH](https://docs.digitalocean.com/products/droplets/how-to/connect-with-ssh/) or use the
+[Droplet Console](https://docs.digitalocean.com/products/droplets/how-to/connect-with-console/).
+
+First upgrade all packages in your VM.
+
+```bash
+apt-get update && apt-get upgrade
+```
+
+The default version of [Node.js](https://nodejs.org/en/) is v10.19.0 on Ubuntu 20.04, we need to install a newer version. You can
+use the following script to install Node.js 16.x on a Ubuntu system.
+
+```bash
+curl -sL https://deb.nodesource.com/setup_16.x -o /tmp/nodesource_setup.sh
+bash /tmp/nodesource_setup.sh
+apt-get install nodejs npm jq unzip
+# Install Node.js 16.x and other dependencies
+```
+
+Confirm the correct version of NodeJs is installed:
+
+```bash
+node -v
+v16.14.2
+```
+
+## Install PostgreSQL
+
+Install PostgreSQL by running the following command:
+
+```bash
+sudo apt install postgresql postgresql-contrib
+sudo -i -u postgres
+```
+
+Create the database:
+
+```bash
+createdb hydra
+```
+
+Change the default password encryption to a stronger one as
+[recommended by PostgreSQL](https://www.postgresql.org/docs/current/auth-password.html):
+
+```bash
+psql
+# Postgres command line
+ALTER SYSTEM SET password_encryption = 'scram-sha-256';
+# Change the default password encryption to stronger one
+SELECT pg_reload_conf();
+# Reload configuration
+```
+
+Let's create a user for Hydra (Use your own password / hash!):
+
+```bash
+CREATE USER hydra PASSWORD 'CHANGE-ME-INSECURE-PASSWORD';
+```
+
+We need to change the `/etc/postgresql/12/main/pg_hba.conf` file and append the following to enable `scram-sha-256` encryption:
+
+```bash
+host    all             all             127.0.0.1/32            scram-sha-256
+```
+
+Check your credentials against PostgreSQL:
+
+```bash
+psql -U hydra -W -h 127.0.0.1
+
+```
+
+Type or paste the password for the Ory Hydra user we created before. If everything works correctly you should see a prompt similar
+to this:
+
+```
+Password:
+psql (12.9 (Ubuntu 12.9-0ubuntu0.20.04.1))
+SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+Type "help" for help.
+
+hydra=>
+```
+
+Congratulations, you have installed and configured the PostgreSQL database. Next we will set up Ory Hydra.
+
+## Install Ory Hydra
```

Comment:
It'd make sense to turn this paragraph into an ordered list to show the flow better.